metron-analytics/metron-profiler/src/main/config/profiler.properties

```diff
@@ -18,21 +18,44 @@
 #
 #
 
-##### Storm #####
+##### Profiler #####
 
-profiler.workers=1
-profiler.executors=0
+# topic from which the profiler consumes messages
 profiler.input.topic=indexing
+
+# how frequently the profiler summarizes data - aka duration of each profile period
 profiler.period.duration=15
 profiler.period.duration.units=MINUTES
+
+# lifespan of a profile - must be greater than twice the profile period
+# intermediate state may be lost if no messages received within this time span
 profiler.ttl=30
 profiler.ttl.units=MINUTES
-profiler.hbase.salt.divisor=1000
+
+# event time processing - the name of the field containing the event timestamp
+profiler.event.timestamp.field=timestamp
+
+# event time processing - how long time lags behind the last seen timestamp
+profiler.event.time.lag=1
+profiler.event.time.lag.units=MINUTES
+
+# where profiles are written to in hbase
 profiler.hbase.table=profiler
 profiler.hbase.column.family=P
+
+# how profiles are written to hbase
+profiler.hbase.salt.divisor=1000
```

Comment:
Perhaps a stupid question, but wouldn't we want the salt divisor in zookeeper so the reader and the writer are both sync'd?  Profiles written with salt divisors of `x` can only be read by clients that use salt divisors of `x`, right?